packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -186,6 +221,21 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
               {tokenSymbol}
             </span>
             <span className="text-xs text-gray-500">{tokenName}</span>
+
+            {isArbitrumToken && (
+              <Tooltip content={arbitrumTokenTooltipContent}>
+                <StatusBadge variant="green">
+                  <CheckCircleIcon className="h-4 w-4" />
+                  <span className="text-xs">Official token</span>
+                </StatusBadge>
+              </Tooltip>
+            )}
+
+            {isPotentialFakeArbitrumToken && (
+              <Tooltip content="This token is different from the official Arbitrum token.">
```

Comment:
NIT: We might need to be more explicit about this design because I completely missed this faint grey error tooltip. It just blends in with the name if we're not being careful.

<img width="449" alt="image" src="https://user-images.githubusercontent.com/7558499/226869226-da6ac7bb-9976-4875-9447-20bd45a9d598.png">

Reply:
Updated the warning color to `brick-dark`, hope it's visible now.